nad/Makefile.am

```diff
@@ -24,7 +24,7 @@ EXTRA_DIST = GL27 nad.lst proj_def.dat nad27 nad83 pj_out27.dist pj_out83.dist t
 		esri.extra other.extra \
 		CH IGNF testIGNF proj_outIGNF.dist \
 		ITRF2000 ITRF2008 ITRF2014 \
-		makefile.vc CMakeLists.txt tests/test_nodata.gtx
+		makefile.vc CMakeLists.txt tests/test_nodata.gtx null.lla
```

Comment:
What's the reasoning behind including it here? The `null` grid is included in the grid distributions.

Reply:
The reasoning is that null.lla is in git, and if we don't had it in EXTRA_DIST it is not part of the generated tarball by make dist. And thus if someone use the tarball and does "make check" without downloading the grids, it won't get the null grid (since 'make' in nad create null from null.lla) and that causes a number of test failures. I think it is reasonable to have the tiny null grid generated by proj itself

Reply:
All right. I agree, it is reasonable to include it.